lib/fetch/sanitize.js

```diff
@@ -155,17 +163,20 @@ const categories = {
 export function sanitizeBadger(json) {
   const get = makeGetter(json);
   const name = get('badger.name');
-  const primaryImageUrl = get('badger.image-url');
-  const secondaryImageUrl = get('badger.secondary-image-url');
+  const sanitizeUrl = (url) => url && url.replace(/^http:/, 'https:');
+  const primaryImageUrl = pathOr(null, ['main', 'url'], get('badger.primary-image'));
+  const secondaryImageUrl = pathOr(null, ['main', 'url'], get('badger.secondary-image'));
+  const fallbackPrimaryImageUrl = get('badger.image-url');
+  const fallbackSecondaryImageUrl = get('badger.secondary-image-url');
```

Comment:
Why supply the fallback as the `defaultValue` (first parameter) of the `pathOr` function
```js
const primaryImageUrl = pathOr(get('badger.image-url'), ['main', 'url'], get('badger.primary-image'));
```
Then you can return the sanitized url without worrying about whether it's the fallback or not
```js
primaryImageUrl: sanitizeUrl(primaryImageUrl),
```
This applies to all the fallback image logic in this file